File: Codewars/CPP/Matrix_creation.cpp
#include <vector>

std::vector <std::vector<int>> getMatrix(const int n) {
    std:: vector <std:: vector <int>> result(n);
    for (int i = 0; i < n; i++)
    {
        std:: vector <int> temp_vector(n);
        for (int j = 0; j < n; j++)
        {
            i == j ? temp_vector[j] = 1 : true;
        }
        result[i] = temp_vector;
    }
    
    return result;
}

File: Codewars/Python/Matrix_creation.py
def get_matrix(n):
    answer = []
    for i in range(n):
        temp_array = []
        for j in range(n):
            if i == j:
                temp_array.append(1)
            else:
                temp_array.append(0)
        answer.append(temp_array)

    return answer